charts/nextcloud/1.6.48/questions.yaml

```diff
@@ -285,6 +285,43 @@ questions:
                       schema:
                         type: hostpath
                         required: true
+        - variable: nextcloud-userdata
+          label: "Nextcloud User Data Volume"
+          schema:
+            type: dict
+            show_if: [[ "nextcloud.datadir", "!=", "/var/www/html/data" ]]
```

Comment:
When the user data remains in the folder `/var/www/html/data`, the additional user data volume should not be created, to support the user hints were added in the variable descriptions. I also wanted to hide the new `Nextcloud User Data Volume` when the default value was not changed by the user. However, the `show_if` directive seems to be misconfigured or being ignored.